app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -274,3 +277,17 @@ class MenuDrawerFragment : MenuFoldersFragment() {
         },
     )
 }
+
+interface MailboxListFragment {
+
+    val mailboxesAdapter: SwitchMailboxesAdapter
+    val currentClassName: String
+
+    fun Fragment.onLockedMailboxClicked(mailboxEmail: String) {
+        safeNavigate(
+            resId = R.id.lockedMailboxBottomSheetDialog,
+            args = LockedMailboxBottomSheetDialogArgs(mailboxEmail).toBundle(),
+            currentClassName = currentClassName,
+        )
+    }
+}
```

Comment:
Move it to its own file.

Reply:
It will be done in a different PR along with all the other interface used twice in the project